### DISCUSSION TITLE
*.client.tsx error when importing with VITE Element type is invalid

### DISCUSSION THREAD

**Discussion by Nedi11** [remix-run/remix#8377] [Q&A] [answered]
I have vite with express server like described in docs. 
I get this error when I import a file that has .client extension on it
`Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

If I add the imported component from .client live reload renders it but when I refresh it breaks

**Comment by gustavopch** [comment]
It will be easier for people to help you if you make a reproduction on StackBlitz: https://stackblitz.com/?starters=fullstack

**Reply by Nedi11** [reply]
https://github.com/remix-run/remix/issues/8378

**Reply by gustavopch** [reply]
Hm, really looks like a bug in Remix.

**Comment by Nedi11** [comment, marked as answer]
https://github.com/remix-run/remix/issues/8378
